out_test.go

```diff
@@ -287,6 +287,23 @@ var _ = Describe("Out", func() {
 				}
 			})
 		})
+
+		Context("When signing image with cosign", func() {
+			BeforeEach(func() {
+				req.Source.Cosign = &resource.Cosign{
+					Registry: "test.registry",
+					Key:      "",
+					Password: "",
+				}
+			})
+
+			It("Signs the image", func() {
+				// there is little to know if the image has been signed
+				// so we assume that if there is no error, then it has been signed
+				Expect(actualErr).ToNot(HaveOccurred())
```

Comment:
We can check the output from the command contains the `logrus.Infof()` message, could we not?
```suggestion
			It("Signs the image", func() {
			Expect(actualErrOutput).To(ContainSubstring("Signing image with Cosign"))
			Expect(actualErrOutput).To(ContainSubstring("Image signed with Cosign"))
				Expect(actualErr).ToNot(HaveOccurred())
```

Reply:
Have added those lines in, although interestingly, even with the cosign key and password being `""` in the test, there are no failures.